cpp/cpp_odbc/Library/src/statement.cpp

```diff
@@ -116,4 +116,12 @@ bool statement::more_results() const
     return do_more_results();
 }
 
+void statement::finalize()
+{
+    if(!finalized){
```

Comment:
please use

    if (not finalized)

to be consistent with the rest of the code base and to be more readable.

Reply:
Will do

Reply:
I actually reverted it now to the prior version using `!`, because the MSVC compiler does not like `not`